HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+
+Further, this project offers great flexibility in terms of team composition, once beyond a requisite minimum.
+The scope for inclusion of mechanically ``solved'' or immensely complicated designs is immense, while the electrical or computer science requirement becomes static after surpassing a threadbare. 
+An electrically gifted team may choose to consider much more advanced imaging equipment or image processing using other factors to control, requiring only a moderate CS investment. 
+Similarly, the potential for much more advanced algorithmic excursions on maintained or minimally modified hardware is substantial. 
+
+As RIT looks to expand its research endeavors, especially in fields such as Imaging Science where it possesses an advantage, developing a long and mutually beneficial relationship with that department is also a goal unto itself. 
+By acting as an experimentalist's platform, SPEX (HAB) has the unique opportunity to be a student body involved in the synthesis, publication and presentation of materials. 
+Be it in academic journals or otherwise, our unique selling point can enable increasing reach and developing leads and future deliverables allowing for assistance and funding from sustainable sources.
+
+\section{Implementation}
+\label{implementation}
+\subsection{Deliverables}
+\label{deliverables}
+\subsubsection{Documentation}
+\label{deliverables-documentation}
+ HAB5 will require GitHub repositories for all software implementations utilized, with documentation utilized referenced. Duplication of online resources, with citations, will be made to aid in reproduction of results or retracing of errors. 
+ Library dependencies are to be mapped. 
+ Electronic components utilized will be logged in a bill of materials (BoM) and KiCad files for any custom PCBs will be archived. 
+ Printed mechanical parts will have files saved and machined components' sketches will be included. 
+ Expectations, hurdles and solutions are to be tabled in documents both before and after the flight.
+
+\subsubsection{Vehicle}
+\label{deliverables-vehicle}
+The desired product at the end of the development is a modular HAB vehicle capable of reliably, predictably, and safely carrying a payload compatible with a n*cubesat size. 
+The vehicle is defined as the mechanically sound HAB structure and common electronic/software components necessary for payload deployment. 
+Mechanical soundness includes, but is not limited to, protection from disintegration, temperature, winds, pressure, charge buildup and vibration. 
+To this end a power system, APRS transmitter, pressure/temperature/humidity sensor, payload arming/disarming mechanism and all facilitating processing and memory elements. 
+There must also be a data analytics system to record flight data and provide information leading to more advanced HABs and missions. 
+This must consist of sensors, processing and memory required for the task.
```

Comment:
What data should be collected? Temperature, pressure, altitude, humidity, etc?

Where would those measurements be made? Internally, externally, somewhere else?